### PR TITLE
Adam Zhao leaderboard submission

### PR DESCRIPTION
**Pull request by ty-zhao1**
Compiled model, mixed precision forward pass with torch.amp.autocast to bfloat16.

I pushed num_layers to 16, d_model -> 768, d_ff -> 2048.

Learning rate range of [1e-6, 1e-3] starting at 1e-5, warming up over 3000 steps and then cooling down to 1e-6 until either 30000 steps have been achieved or the cluster runs out of time (which is what happened).
Batch size 64, weight decay of 0.001, other AdamW parameters set to default.

Tried to do experiment of d_model -> 1024, d_ff -> 2752, but this wouldn't cooperate due to some environment issue.

### CODE REVIEW

**Comment by marcelroed** [comment]
Can you reorder?
